content/en/blog/serverless-twitter-bot-fission/index.md

```diff
@@ -0,0 +1,184 @@
+
+At this point, your Twitter App is ready to be used. Make sure to save `consumer_key`,`consumer_secret`,`access_token`,`access_token_secret`, we will need this later
+
+### Creating a Slack App
+
+1. Head to [Slack API portal](https://api.slack.com/), if you don't have an account create one.
+2. Click on Create A New App and create it from Scratch.
+3. Provide a unique **App Name**.
+4. Select a **Slack Workspace** - _you should be a part of some workspace_
+5. On the basic information page, select **Incoming Webhooks** and activate it.
+6. Click on Add New Webhook to workspace
+7. In a new window, select/enter a new channel name. This is the channel to which your slack app will send notifications.
+8. At this point, you'll have the **Slack webhook url** with you.
+
+{{< figure src="slackbot.gif" alt="Developing Serverless Twitter Bot on Fission" height="600" width="1000">}} Above GIF shows how to create the Slack application.
+
+We have successfully created a Twitter App and a Slack App. Let's now create the application.
+
+## Creating Twitter Bot
+
+You can clone the [TwitterBot repository](https://github.com/fission/examples/tree/master/python/TwitterBot) to create this application. Remember ot update the code with your tokens, keys and usernames.
+
+`app.py` is the file which has the actual code. Make sure to update the `username` in `app.py` before running the script.
+
+As we are using sensitive information like tokens and secrets, we will use **Kubernetes Secrets** to store these and access them in the code
+
+* `consumer_key` - Consumer Key
+* `consumer_secret` - Consumer Secret
+* `access_token` - OAuth Access Token
+* `access_token_secret` - OAuth Access Token Secret
+
+Start by encoding all the keys and secrets
+
+``` bash
+echo -n 'actual-consumer-key' | base64
+EncodedConsumerKey==
+
+echo -n 'actual-consumer-secret' | base64
+EncodedConsumerSecret==
+
+echo -n 'actual-access-token' | base64
+EncodedAccessToken==
+
+echo -n 'actual-access-token-secret' | base64
+EncodedAccessTokenSecret==
+```
+
+Create a new `secrets.yaml` file and add the encoded strings as data. We would be accessing these secrets from our code. Refer to our documentation on [accessing secrets in Fission](https://fission.io/docs/usage/function/access-secret-cfgmap-in-function/) from code.
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: default
+  name: twitter-secret
+data:
+  consumer_key: EncodedConsumerKey==
+  consumer_secret:  EncodedConsumerSecret==
+  access_token: EncodedAccessToken==
+  access_token_secret:  EncodedAccessTokenSecret==
+type: Opaque
+```
+
+Deploy the secret using `kubectl apply -f secrets.yaml`
+
+## Deploying Twitter Bot to Fission
+
+Create a Python environment
+
+```bash
+fission environment create --name python --image fission/python-env --builder fission/python-builder:latest
+```
+
+Create a zip archive as sample.zip archive by executing package.sh script
```

Comment:
There is no mention of source from where it should be taken. 
Only mention of package.sh directly.

Reply:
The package.sh file is in the source code folder itself, hence it if is executed, all the content will be added to archive.

Reply:
can you please mention source code repo and path before execution command ? 
I couldn't infer it from blog.

Reply:
Ok I see its mentioned at top . 

![image](https://user-images.githubusercontent.com/1511975/161675285-ec7ecba8-99b1-4978-9bae-cda1c331c1f9.png)